.travis.yml

```diff
@@ -3,6 +3,7 @@ sudo: false
 script:
   - cargo build --verbose
   - cargo test --verbose
+  - cargo test --verbose --features pattern
   - cargo doc
```

Comment:
I was a little bit torn on that, I'm not sure whether the documentation should describe the "vanilla crate" or the crate with all features enabled. For example if the documentation claims the impl but then I can't use it it may be confusing. I figured that for now the failure mode of "I didn't realize I could use it even though I could" may be more reasonable.